forem/forem
# Fix spacing of Nav Links in Admin

## What type of PR is this? (check all applicable)


## Description
After completing the [Static Nav Sections PR](https://github.com/forem/forem/pull/14292), I noticed that the spacing between the links in Admin was wonky. So this quick PR fixes that.

![Nav Links Spaced Wonky](https://user-images.githubusercontent.com/32520970/128032018-bc1d8448-79e3-45fe-a595-e61a68975750.png)



## QA Instructions, Screenshots, Recordings

Log in as an Admin and check that the nav links on `http://localhost:3000/admin/customization/navigation_links` are spaced nicely.

### UI accessibility concerns?

None impacted

## Added/updated tests?

- [X] No, and this is why: _small UX change that does not necessitate tests_

## [Forem core team only] How will this change be communicated?

      [Admin Guide](https://admin.forem.com/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
- [X] This change does not need to be communicated, and this is why not: _small fix that does not impact any other work_


## [optional] What gif best describes this PR or how it makes you feel?

![Cartoon Character doing quick change](https://media.giphy.com/media/cLS4JxB5VrgRrXoCdd/giphy.gif)
